conf.py

```diff
@@ -31,10 +31,6 @@
 
 #,'runestone.video','runestone.reveal','runestone.poll','runestone.tabbedStuff','runestone.disqus','runestone.codelens','runestone.activecode', 'runestone.assess', 'runestone.animation','runestone.meta', 'runestone.parsons', 'runestone.blockly', 'runestone.livecode']
 
-extension_exclusions = ['runestone.matrixeq', 'runestone.webgldemo']
-for ext in extension_exclusions:
-    extensions.remove(ext)
```

Comment:
Why this change?

Reply:
I would also vote to keep this change.    We have so many js files to load that any little things like this we can do to reduce the number is a good thing.

Reply:
I meant to remove this file and rename the 'old_conf.py' file back to 'conf.py'. I had fetched and merged the latest updates from this repo to my own, but the changes in the conf.py file meant that I wasn't able to properly build the textbook. This is an old version of my conf.py file that still works for me.

Reply:
@laumurph are you using the latest version of Runestone Components?  If its not building for you, please make an issue with the problem so I can track it down.